OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -161,5 +162,20 @@ public void Impact(Target target, Actor firedBy, IEnumerable<int> damageModifier
 					a();
 			}
 		}
+
+		public WDist ModifiedRange(IEnumerable<IRangeModifier> modifiers)
+		{
+			return new WDist(OpenRA.Traits.Util.ApplyPercentageModifiers(Range.Length, modifiers.Select(m => m.GetRangeModifier())));
```

Comment:
This is already `using OpenRA.Traits`, so you can reduce this to `Util.ApplyPercentageModifiers`.